### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by talrejah02**
Screenshot (85)

please review my landing page.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  letter-spacing: 1px;
+  color: var(--primary);
+}
+.text-secondary {
+  margin-top: 1.2rem;
+  max-width: 65rem;
+  text-align: center;
+}
+.text-description {
+  color: #636161;
```
Comment (nshanbhag20001412) [inline style.css:87]: if possible use color variable

**Review by nshanbhag20001412 — APPROVED** [review]
put up a live preview so that it would be easy to review

File: style.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  background-color: var(--primary);
+  color: var(--font-secondary);
+  padding: 0.5rem;
+  z-index: 1;
+  
+  
+}
+.heading {
+  font-size: 24px;
+  line-height: 32px;
```
Comment (himadri2110) [inline style.css:37]: Hey,
You can consider using rems instead of px.
Rest is all good!

**Review by himadri2110 — APPROVED** [review]

**Comment by preethasri** [comment]
looks good and neat

File: style.css (hunk 1 above)
Comment (talrejah02) [reply style.css:87]: will do that, thanks for the review.

File: style.css (hunk 2 above)
Comment (talrejah02) [reply style.css:37]: will do that, thanks for the review.

**Pull request merged**